features/create_an_item.feature

```diff
@@ -8,5 +8,5 @@ Feature: Test post endpoint
     And I create a third Item
     When I request an index of Items
     Then I should see three items
-    And each item should have fields name, quality, sell_in
+    And each item should have fields name, quality, sell_in, description
```

Comment:
The way this is worded implies that the `description` is required. Is that a correct assumption?

Reply:
More generally, in this sort of situation, meaning one where I do not care about the values of the attributes, I would not expect to have to update an acceptance test like this just because I added a new attribute to the model.

I would consider changing this to: `And each item should have the appropriate attributes` or something similar.

Reply:
It will always return a description. This is under "When I request... each item should have fields"

Reply:
for items where no description is given it will still return an empty string